### ISSUE TITLE
Ghost explosion sounds are too loud

### ISSUE THREAD

**Issue by ghost** [UnderscoreKilburn/repentance-issues#1407]
I feel like the explosion sounds from certain ghost items (like Hungry Soul, Purgatory, Vade Retro and Ghost Bombs) are way too loud. The explosion noise is significantly louder than any other sound effects in the game and since they often happen multiple times in a row, I tend to just fully mute my game if I end up with some of those items.
It would be great if the volume of these effects could be reduced.

**Comment by Lonrite** [comment]
same kinda can be said about technology 1 or smth along those lines i think, it's rly loud but i got used to it at this point because i rarely play with in-game music nowadays

**Comment by ghost** [comment]
> same kinda can be said about technology 1 or smth along those lines i think, it's rly loud but i got used to it at this point because i rarely play with in-game music nowadays

Agreed, most laser sound effects are also way too loud. I'm using a mod called Quiet Lasers which tones the volume of those down, but it would be great if those could be made a little more quiet too.
